internal_metrics/dimensions.yaml

```diff
@@ -0,0 +1,74 @@
+line:
```

Comment:
can/should we list for which internal metric(s) each dimension may exist?  For example, I think "line" and "filename" are only applicable to bpf_log and codetiming_* internal metrics.

And/or should we skip documenting the full set of dimensions for low-level debugging type internal metrics such as bpf_log and codetiming_*?

Reply:
I do not the answer. Also it doesn't make sense for field or whoever to need all these knowledge. They can't debug it for us.